### ISSUE TITLE
MQTT adapter: NPE when processing command response

### ISSUE THREAD

**Issue by calohmn** [eclipse/hono#2465]
The following NPE was observed in the logs:
````
java.lang.NullPointerException: null
	at java.base/java.util.Objects.requireNonNull(Unknown Source)
	at org.eclipse.hono.service.AbstractProtocolAdapterBase.isAdapterEnabled(AbstractProtocolAdapterBase.java:531)
	at org.eclipse.hono.adapter.mqtt.AbstractVertxBasedMqttProtocolAdapter.uploadCommandResponseMessage(AbstractVertxBasedMqttProtocolAdapter.java:1174)
	at org.eclipse.hono.adapter.mqtt.AbstractVertxBasedMqttProtocolAdapter.uploadMessage(AbstractVertxBasedMqttProtocolAdapter.java:982)
	at org.eclipse.hono.adapter.mqtt.impl.VertxBasedMqttProtocolAdapter.lambda$onPublishedMessage$2(VertxBasedMqttProtocolAdapter.java:87)
	at io.vertx.core.Future.lambda$compose$3(Future.java:368)
	at io.vertx.core.impl.FutureImpl.dispatch(FutureImpl.java:105)
	at io.vertx.core.impl.FutureImpl.onComplete(FutureImpl.java:83)
	at io.vertx.core.Future.compose(Future.java:364)
	at io.vertx.core.Future.compose(Future.java:336)
	at org.eclipse.hono.adapter.mqtt.impl.VertxBasedMqttProtocolAdapter.onPublishedMessage(VertxBasedMqttProtocolAdapter.java:79)
	at org.eclipse.hono.adapter.mqtt.AbstractVertxBasedMqttProtocolAdapter.lambda$handlePublishedMessage$50(AbstractVertxBasedMqttProtocolAdapter.java:889)
	at io.vertx.core.Future.lambda$compose$3(Future.java:368)
	at io.vertx.core.impl.FutureImpl.dispatch(FutureImpl.java:105)
	at io.vertx.core.impl.FutureImpl.onComplete(FutureImpl.java:83)
	at io.vertx.core.Future.compose(Future.java:364)
	at io.vertx.core.Future.compose(Future.java:336)
	at org.eclipse.hono.adapter.mqtt.AbstractVertxBasedMqttProtocolAdapter.handlePublishedMessage(AbstractVertxBasedMqttProtocolAdapter.java:889)
	at org.eclipse.hono.adapter.mqtt.AbstractVertxBasedMqttProtocolAdapter.lambda$registerHandlers$74(AbstractVertxBasedMqttProtocolAdapter.java:1569)
	at io.vertx.mqtt.impl.MqttEndpointImpl.handlePublish(MqttEndpointImpl.java:528)
	at io.vertx.mqtt.impl.MqttServerConnection.handlePublish(MqttServerConnection.java:329)
	at io.vertx.mqtt.impl.MqttServerConnection.handleMessage(MqttServerConnection.java:143)
````

**Comment by calohmn** [comment]
Occurs when the tenant object isn't already in the cache.

**Comment by calohmn** [comment]
Fix also applied to 1.5.x and 1.6.x.
